graphql_query_derive/src/query.rs

```diff
@@ -71,6 +75,12 @@ impl QueryContext {
             ));
         }
 
+        self.variables_derives.extend(
+            attribute_value
+                .split(',')
+                .map(|s| s.trim())
+                .map(|s| Ident::new(s, Span::call_site())),
+        );
```

Comment:
So we derive the same traits as for the response at the moment, that seems like a good start. I think we might want to enable users to choose different derives for variables and ResponseData (since the use cases are different) in the future, but it's a good start. Great!